.github/dependabot.yml

```diff
@@ -21,3 +21,6 @@ updates:
       python:
         patterns:
           - "*"
+    ignore:
+        # TODO: ignore all updates for JAX GPU due to cuda version issue
```

Comment:
Is this a todo or just a note? Also we should add a newline at end of file.

Reply:
Added a newline and also removed TODO and just kept the note.  Will merge the PR